charts/federation-v2/Chart.yaml

```diff
@@ -1,5 +1,4 @@
 apiVersion: v1
-appVersion: 0.0.2
```

Comment:
any comments to remove this?

Reply:
Yes, it's in the first comment: `Remove appVersion from chart yaml to avoid having to maintain it`

Currently we don't have a reason to set an `appVersion` separate from the chart version, and `appVersion` is optional, so I figured it was preferable to remove it rather than having to update it with each release.